python/tests/MNVMerge_test.py

```diff
@@ -0,0 +1,176 @@
+#   LICENSE
+# Copyright (c) 2018-2019 Genome Research Ltd.
+# Author: Cancer Genome Project cgphelp@sanger.ac.uk
+#
+#
+# This file is part of vcf_flag_modifier.
+#
+# vcf_flag_modifier is free software: you can redistribute it and/or modify it under
```

Comment:
Wrong licence, years and program, skywalkingeyes will find this for you.

Reply:
Added skywalkingeyes to  CI checks.